.travis.yml

```diff
@@ -52,3 +28,10 @@ jobs:
       before_script: skip
       script:
         - mvn test -B
+    - stage: report_coverage
+      name: report_coverage
+      # Generating test coverage report and publishing to Codacy
+      script:
+        - mvn cobertura:cobertura -B
+        - curl -Ls -o codacy-coverage-reporter-assembly.jar https://github.com/codacy/codacy-coverage-reporter/releases/download/6.0.2/codacy-coverage-reporter-6.0.2-assembly.jar
```

Comment:
Why pin it to 6.0.2? To get the latest take a look at https://github.com/eiffel-community/eiffel-gerrit-plugin/blob/master/.travis.yml

Reply:
I always want to pin a version that I know works and not risk having an update that may break or need a new version of for example coberura. I never uses latest.

Reply:
Why 6.0.2 and not 6.0.6?

Reply:
Have not tested with 6.0.6.

Reply:
After investigating release log from 6.0.2 to 6.0.6 I see no need to update to 6.0.6 since it would give no value.

Reply:
Asked as the recomendations on https://github.com/codacy/codacy-coverage-reporter#github is used for the gerrit plugin.

Comment resolved.